_posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: 'Quarkus 3.0.0.Alpha5 released - now with Hibernate 6, new Developer Console and more!'
+date: 2023-02-18
+tags: release
+synopsis: 'This version is the second biggest update with major impact for Quarkus 3 - upgrading Hibernate 5 to Hibernate 6.'
+author: maxandersen
+---
+
+We are in the process of getting Quarkus 3 ready - last we got our main branch to Jakarta EE 10 to let us getting more things integrated and tested faster.
+
+The big one this time is the move from Hibernate 5 to Hibernate 6 - which *will* cause breakages thus please read the section below for details if you are Hibernate ORM or Hibernate Reactive user.
+
+  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & Hibernate Reactive 2
```

Comment:
Not sure the PRs would be very useful to application developers, especially such large PRs...

In any case, if you want those links, you could add the HR one:

```suggestion
  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & https://github.com/quarkusio/quarkus/pull/31454[Hibernate Reactive 2]
```